web/src/components/apps/AppConfig.jsx

```diff
@@ -412,7 +417,7 @@ class AppConfig extends Component {
     if (!match.params.sequence) return false;
     const sequence = parseInt(match.params.sequence);
     const isCurrentVersion = app.downstreams[0]?.currentVersion?.sequence === sequence;
-    const isLatestVersion = app.currentVersion.sequence === sequence;
+    const isLatestVersion = app.downstreams[0]?.pendingVersions[0]?.sequence === sequence;
```

Comment:
pendingVersions could be empty
```suggestion
    const isLatestVersion = app.downstreams[0]?.pendingVersions?.length && app.downstreams[0]?.pendingVersions[0]?.sequence === sequence;
```